slate/source/openapi/swagger-initializer.js

```diff
@@ -4,16 +4,16 @@ window.onload = function() {
     // the following lines will be replaced by docker/configurator, when it runs in a docker-container
     window.ui = SwaggerUIBundle({
       urls: [
-        { name: "alerts", url: "./specs/alerts-rest.json"},
+        { name: "alerts", url: "./specs/alerts.json"},
         // { name: "biocache", url: "./specs/biocache-http.yaml"},
         // { name: "ecodata", url: "./specs/ecodata-http.yaml"},
-        // { name: "images", url: "./specs/images-rest.yaml"},
-        { name: "userdetails", url: "./specs/userdetails-rest.json"},
-        { name: "logger", url: "./specs/logger-rest.json"},
-        { name: "doi", url: "./specs/doi-rest.json"},
-        { name: "biocollect", url: "./specs/biocollect-rest.json"},
-        { name: "bie-index", url:"./specs/bie-index-rest.json"},
-        { name: "specieslist", url:"./specs/specieslist-rest.json"}
+        { name: "images", url: "./specs/images.json"},
```

Comment:
Is this a recent change ? Will all the branch in api gateway repo need to have their app_setup configs updated as well? e.g. 
`REST_OPENAPI_EXPORT_FILENAME=logger-rest.json`

Reply:
Hi Sushant, yes, there is an update on how to export the api spec. The new change is to grab the api spec from the branch itself and copy it to s3 bucket. I will update the documentation.